pkg/kubelet/eviction/helpers_test.go

```diff
@@ -1645,95 +1645,6 @@ func TestGetStarvedResources(t *testing.T) {
 	}
 }
 
-func testParsePercentage(t *testing.T) {
```

Comment:
I think those two actually should be real tests.

Can you please change it's name to:
TestParsePercentage
(similarly below)
so that they will actually be run as tests?